lib/pages/home/wallet/wallet.dart

```diff
@@ -57,12 +62,19 @@ class WalletPage extends StatelessWidget {
     );
   }
 
-  Widget _buildCurrencyRow(BuildContext context, Currency currency) {
+  Widget _buildCurrencyRow(BuildContext context, int index, Currency currency) {
+    Color currencyRowColour = Colors.white12;
+    Color highlightColor = Theme.of(context).brightness ==  Brightness.light ? Colors.black12 : Colors.white12;
+    currencyRowColour = (index % 2 != 0 ? currencyRowColour = highlightColor : currencyRowColour = Colors.transparent);
     return Container(
-      margin: EdgeInsets.all(8.0),
+      margin: EdgeInsets.all(0.0),
+      padding: EdgeInsets.only(left: 0.0, right: 8.0, top: 8.0, bottom: 8.0),
```

Comment:
0 values when using EdgeInsets are unnecessary. You could remove those and change it to:
```
padding: EdgeInsets.only(top: 8.0, bottom: 8.0),
```
However, you can make it look cleaner by using symmetric:
```
padding: EdgeInsets.symmetric(vertical: 8.0),
```

Reply:
On second look, I'd rather have you make this `padding: EdgeInsets.all(8.0),` as there is no padding on the right side at the moment.

Reply:
Updated.